app/controllers/events_controller.rb

```diff
@@ -1,3 +1,10 @@
 class EventsController < ApplicationController
-  def index; end
-end
+  def index
+    @events = Event.all
+  end
+
+  def show
```

Comment:
Checking the official rails documentation, the show method should be used to display the detail of an object, not the list of it, please consider following good practices andmaking this change, attached documentation
[Rails documentation](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions)